### PR TITLE
Update dockerfile CRIU install

### PR DESCRIPTION
**Pull request by LongyuZhang**
- update CRIU branch to 0.44 releaase
- add option CONFIG_AMDGPU=n
- Related Issue: runtimes/backlog/1380

### CODE REVIEW

**Comment by LongyuZhang** [comment]
Test Links:
```
15:36:48       [exec] STEP 6/25: RUN mkdir -p /tmp 	&& cd /tmp 	&& git clone https://github.com/ibmruntimes/criu.git 	&& cd criu 	&& git fetch origin 	&& git reset --hard origin/0.44.0-release 	&& make PREFIX=/usr CONFIG_AMDGPU=n install 	&& criu -V
15:36:49       [exec] Cloning into 'criu'...
...
15:37:52       [exec] Version: 3.18
15:37:52       [exec] GitID: 22bedc9d1
15:37:55       [exec] --> 17a29055524
15:37:55       [exec] STEP 7/25: RUN setcap 
```
x86-64 linux: [38667](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/38667/)
ppcle-linux: [38669](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/38669/)
s390x: [38668](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/38668/)

**Review by karianna — APPROVED** [review]

File: external/dockerfile_functions.sh [hunk 1]
```diff
@@ -397,8 +397,8 @@ print_criu_install() {
                 "\n\t&& git clone https://github.com/ibmruntimes/criu.git \\" \
                 "\n\t&& cd criu \\" \
                 "\n\t&& git fetch origin \\" \
-                "\n\t&& git reset --hard origin/0.43.0-release \\" \
-                "\n\t&& make PREFIX=/usr install \\" \
+                "\n\t&& git reset --hard origin/0.44.0-release \\" \
+                "\n\t&& make PREFIX=/usr CONFIG_AMDGPU=n install \\" \
```
Comment (karianna) [inline external/dockerfile_functions.sh:401]: Is that always true?  That there's no AMDGPU involved.

Comment (LongyuZhang) [reply external/dockerfile_functions.sh:401]: Currently the AMD GPU module built has some bugs, so disable it for now.

Comment (JasonFengJ9) [reply external/dockerfile_functions.sh:401]: Assuming this conversation can be resolved.

**Review by JasonFengJ9 — APPROVED** [review]

**Pull request merged**